plugins/frontend/backstage-plugin-github-insights/.eslintrc.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
   extends: [require.resolve('@backstage/cli/config/eslint')],
+  rules: {
+    'notice/notice': 'off',
```

Comment:
why off ? this is public, we want this no?

Reply:
I think we set it like this in all of the plugins, we use eslintrc in root. This worked strange as it reported errors even when notice was added